### DISCUSSION TITLE
Unable to find log files in version 2.1.1

### DISCUSSION THREAD

**Discussion by jaybaeer** [postalserver/postal#1929] [Help with using Postal]
I can't find log files, i searched in /opt/postal/, adn in /var/log/ but nothing found.

**Comment by willpower232** [comment]
Most of the logs are put to stdout now so you can see their contents with the `postal logs` command (i.e. `postal logs smtp`)

The only exception to this is the web ones where any problems with rails are written to a log file in the container.
